tests/utils/test_text_atlas.py

```diff
@@ -77,7 +77,7 @@ def test_atlas_resize():
     prev_array = atlas._array
 
     # We can fit 9 regions of 8x8 in it. So 4 more.
-    for i in range(4):
+    for _i in range(4):
```

Comment:
hmm i didn't know this trick, i always struggled with `_` since it didn't give any info as to what the variable was.

Reply:
I just did what ruff suggested 🤷